public/js/components/card-drop-down.jsx

```diff
@@ -20,15 +20,51 @@ export default class CardDropDown extends Component {
     ).isRequired,
     cssModifier: PropTypes.string,
     onCardChange: PropTypes.func.isRequired,
+    selectedCardResource: PropTypes.string,
+    showDefaultOption: PropTypes.bool.isRequired,
+  }
+
+  static defaultProps = {
+    showDefaultOption: true,
   }
 
   constructor(props) {
     super(props);
-    this.state = {
+    var cardData;
+    // Get the data for a the default selected card.
+    if (props.selectedCardResource) {
+      cardData = this.getSelectedCard(props.selectedCardResource);
+    }
+
+    // If there's default selected option and the defaultOption is off.
+    // we set the "fakeSelect" to the first option.
+    if (!cardData && !props.defaultOption) {
+      cardData = props.cards[0];
+    }
+
+    var initialState = {
       selectedText: defaultSelectText,
       selectedCardType: null,
       isFocused: false,
     };
+
+    if (cardData) {
+      initialState.selectedText = '●●●● ●●●● ●●●● ' + cardData.truncated_id;
+      initialState.selectedCardType = cardData.type_name.toLowerCase();
+    }
```

Comment:
I think it would be cleaner if a lot of this default selection code were to reuse `updateCardSelection()` even if it means pulling something out of there into a helper method.